.github/workflows/build_frontend.yaml

```diff
@@ -60,5 +60,9 @@ jobs:
 
       - name: Build staging fallthrough for PR branch (conditional check)
         if: (!(github.ref == 'refs/heads/master' || github.ref == 'refs/heads/main' || github.ref == 'refs/heads/production'))
+        working-directory: ./frontend-react
         run: |
+          yarn lint:prettier
```

Comment:
@JosiahSiegel for awareness regarding https://github.com/CDCgov/prime-reportstream/pull/3392